app/models/account.rb

```diff
@@ -110,6 +129,13 @@ def self.with_orders_for_facility(facility)
     where(id: ids_with_orders(facility))
   end
 
+  # The subclassed Account objects will be cross facility by default; override
+  # this method if the subclassed Account object is always scoped to a
+  # particular facility.
+  def facility
```

Comment:
Perhaps the comment should say something about overriding using `belongs_to :facility`. That's how we use do in CC/POs.